tests/admin/test_view.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for the `edx-enterprise` admin forms module.
+"""
+from __future__ import absolute_import, unicode_literals
+
+import json
+
+import ddt
+import six
+from pytest import mark
+
+from django.conf import settings
+from django.test import Client, TestCase, override_settings
+
+from enterprise import admin as enterprise_admin
+from enterprise.admin import EnterpriseCustomerManageLearnersView
+from enterprise.django_compatibility import reverse
+from enterprise.models import EnterpriseCustomerUser
+from test_utils.factories import FAKER, EnterpriseCustomerFactory, EnterpriseCustomerUserFactory, UserFactory
+
+
+@ddt.ddt
+@mark.django_db
+@override_settings(ROOT_URLCONF="test_utils.admin_urls")
+# TODO: remove suppression when https://github.com/landscapeio/pylint-django/issues/78 is fixed
+# pylint: disable=no-member
+class TestEnterpriseCustomerManageLearnersView(TestCase):
+    """
+    Tests EnterpriseCustomerManageLearnersView.
+    """
+    def setUp(self):
+        """
+        Test set up - installs common dependencies.
+        """
+        super(TestEnterpriseCustomerManageLearnersView, self).setUp()
+        self.user = UserFactory(is_staff=True, is_active=True)
+        self.user.set_password("QWERTY")
+        self.user.save()
+        self.enterprise_customer = EnterpriseCustomerFactory()
+        self.default_context = {
+            'has_permission': True,
+            'opts': self.enterprise_customer._meta,
+            'user': self.user
+        }
+        self.view_url = reverse(
+            'admin:' + enterprise_admin.utils.UrlNames.MANAGE_LEARNERS,
+            args=(self.enterprise_customer.uuid,)
+        )
+        self.client = Client()
+        self.context_parameters = EnterpriseCustomerManageLearnersView.ContextParameters
+
+    def _test_common_context(self, actual_context, context_overrides=None):
+        """
+        Test common context parts.
+        """
+        expected_context = {}
+        expected_context.update(self.default_context)
+        expected_context.update(context_overrides or {})
+
+        for context_key, expected_value in six.iteritems(expected_context):
+            assert actual_context[context_key] == expected_value
```

Comment:
@zubair-arbi I find it really strange using `py.test` for test runner and not using anything else, especially smart assertions - the [top feature](http://doc.pytest.org/en/latest/#id1) as listed by pytest developers/documentation team.